README.md

```diff
@@ -1,2 +1,22 @@
 # bootstrap
 Foundation Bootstrap Team
+
+## What's in a name?
+
+As we do not yet have a name for a merged entity we shall refer to merged entity in this document
+as $FOUNDATION
+
+## Mission
+
+$FOUNDATION envisions a future where JavaScript is a sustainable platform for implementing best in
```

Comment:
Does this correspond to what was discussed at the summit?
«envisions a future where …» part seems pretty controversial to me.

Reply:
Not entirely. In the afternoon session we worked through an exercise of creating several possible mission statements. We concluded that each of those should be posted as work-in-progress statements and that we would coallesce into one over time. I think those statements should be added here with clear language indicating work in progress.